broadinstitute/single_cell_portal_core
# Remove the image upload step from the upload wizard (SCP-5045)

Remove the unsupported reference image upload step in the upload wizard and accompanying feature flag references. 
Also remove the units from the AnnData expression matrix tab in preparation for Raw Counts being added as a full option for AnnData upload in [SCP-4754 ](https://broadworkbench.atlassian.net/browse/SCP-4754?atlOrigin=eyJpIjoiZmFiYjA3YmQ0ZGYxNGFjYTgwMDEzMTgxOTU4ZmYzMmEiLCJwIjoiaiJ9)

To test
- pull this branch and run the db migration via `bin/rails db:migrate`
- then start up a local server and navigate to the feature flags page: https://localhost:3000/single_cell/feature_flags and see that `reference_image_upload` is not there.
- navigate to the upload wizard on any study and note that the image tab is not there.
